ModuleNotFoundError: Module not found: Error: [CaseSensitivePathsPlugin] SessionStore.ts

## Description
Running `npm start` causes this error regarding:
`[webpack-dev-middleware] ModuleNotFoundError: Module not found: Error: [CaseSensitivePathsPlugin]`
for:
- `SessionStore.ts`
- `ExtensionStore.ts`
- `DevStore.ts`
- `UserScheduleStore.ts`

--- a/README.md
+++ b/README.md
@@ -3,12 +3,12 @@
 ## Built Using:
 
 -   React 18
--   Typescript
+-   TypeScript
 -   Webpack 5 (esbuild-loader)
--   Eslint
+-   ESLint
 -   Prettier
 -   Semantic-Release
--   Custom Messaging & Storage wrappers
+-   Custom Messaging & Storage Wrappers
 
 ## Getting Started
 

--- a/src/background/background.ts
+++ b/src/background/background.ts
@@ -5,7 +5,7 @@ import onInstall from './events/onInstall';
 import onNewChromeSession from './events/onNewChromeSession';
 import onServiceWorkerAlive from './events/onServiceWorkerAlive';
 import onUpdate from './events/onUpdate';
-import { sessionStore } from '../shared/storage/sessionStore';
+import { sessionStore } from '../shared/storage/SessionStore';
 import browserActionHandler from './handler/browserActionHandler';
 import hotReloadingHandler from './handler/hotReloadingHandler';
 import tabManagementHandler from './handler/tabManagementHandler';

--- a/src/background/events/onInstall.ts
+++ b/src/background/events/onInstall.ts
@@ -1,4 +1,4 @@
-import { extensionStore } from '../../shared/storage/extensionStore';
+import { extensionStore } from '../../shared/storage/ExtensionStore';
 
 /**
  * Called when the extension is first installed or synced onto a new machine

--- a/src/background/events/onUpdate.ts
+++ b/src/background/events/onUpdate.ts
@@ -1,5 +1,5 @@
 import { hotReloadTab } from 'src/background/util/hotReloadTab';
-import { extensionStore } from '../../shared/storage/extensionStore';
+import { extensionStore } from '../../shared/storage/ExtensionStore';
 
 /**
  * Called when the extension is updated (or when the extension is reloaded in development mode)

--- a/src/background/handler/hotReloadingHandler.ts
+++ b/src/background/handler/hotReloadingHandler.ts
@@ -1,6 +1,6 @@
 import HotReloadingMessages from 'src/shared/messages/HotReloadingMessages';
 import { MessageHandler } from 'chrome-extension-toolkit';
-import { devStore } from 'src/shared/storage/devStore';
+import { devStore } from 'src/shared/storage/DevStore';
 
 const hotReloadingHandler: MessageHandler<HotReloadingMessages> = {
     async reloadExtension({ sendResponse }) {

--- a/src/background/lib/addCourse.ts
+++ b/src/background/lib/addCourse.ts
@@ -1,4 +1,4 @@
-import { userScheduleStore } from 'src/shared/storage/userScheduleStore';
+import { userScheduleStore } from 'src/shared/storage/UserScheduleStore';
 import { Course } from 'src/shared/types/Course';
 
 /**

--- a/src/background/lib/removeCourse.ts
+++ b/src/background/lib/removeCourse.ts
@@ -1,4 +1,4 @@
-import { userScheduleStore } from 'src/shared/storage/userScheduleStore';
+import { userScheduleStore } from 'src/shared/storage/UserScheduleStore';
 import { Course } from 'src/shared/types/Course';
 
 /**

--- a/src/background/util/hotReloadTab.ts
+++ b/src/background/util/hotReloadTab.ts
@@ -1,4 +1,4 @@
-import { devStore } from 'src/shared/storage/devStore';
+import { devStore } from 'src/shared/storage/DevStore';
 
 /**
  * A list of websites that we don't want to reload when the extension reloads (becuase it'd be hella annoying lmao)

--- a/src/background/util/openDebugTab.ts
+++ b/src/background/util/openDebugTab.ts
@@ -1,4 +1,4 @@
-import { devStore } from 'src/shared/storage/devStore';
+import { devStore } from 'src/shared/storage/DevStore';
 
 /**
  * Open the debug tab as the first tab

--- a/src/debug/index.tsx
+++ b/src/debug/index.tsx
@@ -1,7 +1,7 @@
 import './hotReload';
 import React, { useEffect } from 'react';
 import { render } from 'react-dom';
-import { devStore } from 'src/shared/storage/devStore';
+import { devStore } from 'src/shared/storage/DevStore';
 
 const manifest = chrome.runtime.getManifest();
 

--- a/src/shared/storage/UserScheduleStore.ts
+++ b/src/shared/storage/UserScheduleStore.ts
@@ -20,5 +20,4 @@ export const userScheduleStore = createLocalStore<IUserScheduleStore>({
     ],
 });
 
-
 debugStore({ userScheduleStore });

--- a/src/views/hooks/useUserSchedules.ts
+++ b/src/views/hooks/useUserSchedules.ts
@@ -1,6 +1,6 @@
 import { Serialized } from 'chrome-extension-toolkit';
 import { useEffect, useState } from 'react';
-import { userScheduleStore } from 'src/shared/storage/userScheduleStore';
+import { userScheduleStore } from 'src/shared/storage/UserScheduleStore';
 import { UserSchedule } from 'src/shared/types/UserSchedule';
 
 export default function useUserSchedules(): UserSchedule[] {

